docs/External_HSS_Integration.md

```diff
@@ -71,6 +71,26 @@ In order to register and make calls, you need to create subscriber records on yo
 2. create an associated private user identity with its ID formed by removing the `sip:` scheme prefix from the public user ID
 3. configure the public user identity's Initial Filter Criteria to include an application server named `sip:mmtel.your.home.domain`, where `your.home.domain` is replaced with your home domain - this enables MMTEL services for this subscriber.
 
+### Allowing one subscriber to have two private identities
+
+If you wish to use a stock Android SIP client that doesn't contain an
```

Comment:
I think you could make this a bit more generic - this applies to any SIP client that doesn't allow the username to be configured, and defaults to a username like `1234` rather than `1234@example.com`.